tests/unit/test_charm_2.py

```diff
@@ -1,12 +1,48 @@
+import os
 import unittest
 import unittest.mock
 
 import ops.testing
 from charm import WordpressCharm
 
+METADATA_FILE = os.path.join(os.path.split(__file__)[0], "..", "..", "metadata.yaml")
```

Comment:
What about `Pathlib`: https://docs.python.org/3/library/pathlib.html, it is a higher level API which I find very easy to read and understand

Reply:
Yes, definitely `Pathlib`, but this line here is currently not used, so I will just remove this for now, thanks!

Reply:
Thats even better!